src/quantum/widget.js

```diff
@@ -62,7 +62,6 @@ export default class Widget extends React.Component {
         </aside>
       );
     }
-
```

Comment:
Was this deliberate? Prefer not to make whitespace-only changes as part of a larger pr like this.

Reply:
Nope, I'll remove that with the runaway console.log's - thanks!